cpp/models/abm/world.h

```diff
@@ -447,6 +470,75 @@ class World
     }
     /** @} */
 
+    /**
+     * @brief Flip activeness status of a person in the world.
+     * @param[in] person_id PersonId of Person whose activeness status is fipped.
+     */
+    void change_activeness(PersonId person_id)
+    {
+        m_activeness_statuses[person_id.get()] = !m_activeness_statuses[person_id.get()];
+    }
+
+    /**
+     * @brief Copy the persons from another World to this World. 
+     * If the persons are at a location in this world they are activated, otherwise they are deactivated.
+     * If necessary the person ids are changed such that they correspond to the index in this world's m_persons vector.
+     * @param[in] other The World the Person%s are copied from.
+     */
+    void copy_persons_from_other_world(const World& other)
```

Comment:
I would prefer adding an (optional) world_id to World::add_person and make this an external function using add_person.